### DISCUSSION TITLE
我们是否可以借助Dhikuzu来启动shikuzu

### DISCUSSION THREAD

**Discussion by JOGIRAFFER** [RikkaApps/Shizuku#576] [Ideas]
emmmm,刚刚是拿英文写的，有些烂，所以我决定拿中文再发一遍请求。
![Screenshot_20240813_121710_com.android.settings.jpg](https://github.com/user-attachments/assets/98eca1ba-7446-4f6e-b29a-9b71c62e0465)

非常巧，华为和荣耀的老机子都搞不了Shikuzu无线连接，后者是因为android10没有，前者则是因为harmonyOS把相关功能隐藏了🤣

所以，我做不到无线连接，加上麒麟芯片的强大魅力，我什么都搞不了。😂直到我发现了Dhikuzu
![Screenshot_20240813_115300_com.huawei.android.launcher.png](https://github.com/user-attachments/assets/aef59fe6-6bda-49c9-a977-2de0452722ee)

一款借由shikuzu搭建的软件，提供的是设备所有者权限![Screenshot_20240813_115707_com.rosan.dhizuku.jpg](https://github.com/user-attachments/assets/e9615079-9002-426f-a783-0d3b1d44d943)

可以永久激活，很妙。

缺点也很大，![Screenshot_20240813_122214_com.rosan.dhizuku.jpg](https://github.com/user-attachments/assets/14ceb681-88ab-462e-990d-e76f15db8d54)

据我所知，除了OwnDroid没软件能用.
如果要用，也得借助各种Patch注入Dax插件，但问题是注入之后容易死完全无法启动。

所以，我想到了一种可能性：是否可以借助Dhikuzu提供的设备所有者权限来永久激活Shikuzu

如果可以的话能改一下吗QAQ

**Comment by banjuanshu0007** [comment]
问题解决了吗？华为手机没有无线调试，可以安装shizuku吗？
